src/common/gui/CLFOGui.cpp

```diff
@@ -1180,16 +1184,34 @@ CMouseEventResult CLFOGui::onMouseMoved(CPoint& where, const CButtonState& butto
             float f = (float)(steprect[i].bottom - where.y) / steprect[i].getHeight();
             if (buttons & (kControl | kRButton))
                f = 0;
-            else if (lfodata->unipolar.val.b)
-               f = limit_range(f, 0.f, 1.f);
             else
-               f = limit_range(f * 2.f - 1.f, -1.f, 1.f);
-            if ( (buttons & kShift) )
+               if (lfodata->unipolar.val.b)
```

Comment:
why did this change from an else if to an if? Seems wrong no? (This is why I wish the codebase used curly braces everywhere btw :) )